server/migrations/20250426220809_init.sql

```diff
@@ -33,6 +33,7 @@ CREATE TABLE project_versions (
 CREATE TABLE tasks (
     id int8 GENERATED ALWAYS AS IDENTITY NOT NULL PRIMARY KEY,
     created_at timestamptz NOT NULL DEFAULT now(),
+    deadline interval NOT NULL DEFAULT '3 days'::interval,
```

Comment:
I'm not 100% sure about having a default. I like it personally for short-handing things, but on the other hand, it's possible someone won't know to set it without receiving the error telling them they forgot it.

Maybe 3 days is far too short for something, or far too long for something else.

Reply:
removing default for now